aries_cloudagent/config/argparse.py

```diff
@@ -417,6 +417,14 @@ def add_arguments(self, parser: ArgumentParser):
                 "Default: false."
             ),
         )
+        parser.add_argument(
+            "--cred-issue-ack-required",
+            action="store_true",
+            env_var="ACAPY_CRED_ISSUE_ACK_REQUIRED",
```

Comment:
This raises another question -- how to define configuration settings for when to request ACKs?  For example, is there a way with ArgParser to add a parameter type `--ack-request-protocol <protocol>` that can be defined multiple times?  I don't know if that is even possible...

Likewise, should the Admin API be extended for supported protocols to add a `add-please-ack` on certain endpoints? That would mean that the controller would be responsible for using please ack when necessary/desirable.